src/main/fc/fc_msp.c

```diff
@@ -1845,6 +1843,7 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
 #ifndef USE_QUAD_MIXER_ONLY
     case MSP_SET_MIXER:
         mixerConfigMutable()->mixerMode = sbufReadU8(src);
+        mixerUpdateStateFlags();    // Required for correct preset functionality
```

Comment:
@digitalentity Almost, this has to be called also in MSP_SET_BF_CONFIG
Configurator is using this frame instead of MSP_SET_MIXER